wrappers/golang/curves/bls12377/bls12_377.go

```diff
@@ -0,0 +1,4 @@
+package bls12377
```

Comment:
Its there to allow a single time of linking with the library instead of having the linking command in each file that uses it. Other than that nothing really